lib/apiservers/portlayer/restapi/configure_port_layer.go

```diff
@@ -163,3 +167,38 @@ func setupMiddlewares(handler http.Handler) http.Handler {
 func setupGlobalMiddleware(handler http.Handler) http.Handler {
 	return handler
 }
+
```

Comment:
exported function might need a brief comment to explain what it does, like ByteStreamProducer.